src/components/exercise5/list.jsx

```diff
@@ -0,0 +1,11 @@
+export default function List() {
+  const animals = ["dog", "cat", "chicken", "cow", "sheep", "horse"];
+  let animalList = animals.map((animal, index) => (
+    <li key={index}>{animal}</li>
```

Comment:
using index as a key is an anti-pattern. It may break your application and display wrong data!
[this link can be helpful](https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/)